tutorials/algorithms/07_grover_examples.ipynb

```diff
@@ -11,17 +11,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pylab\n",
     "import numpy as np\n",
-    "from qiskit import Aer\n",
-    "from qiskit.utils import QuantumInstance\n",
-    "from qiskit.tools.visualization import plot_histogram\n",
+    "import pylab\n",
+    "\n",
     "from qiskit.algorithms import Grover, AmplificationProblem\n",
-    "from qiskit.circuit.library.phase_oracle import PhaseOracle"
+    "from qiskit.circuit.library.phase_oracle import PhaseOracle\n",
+    "from qiskit.primitives import Sampler\n",
+    "from qiskit.tools.visualization import plot_histogram"
```

Comment:
I'd recommend to have the imports in the cell they are used. That way a user can understand better why they are importing and where it's needed.

Reply:
Thank you for the comment. I moved the imports in the cells they are used.